include/bhc/bhc.hpp

```diff
@@ -45,11 +45,15 @@ namespace bhc {
  * params, outputs: Just create uninitialized structs and pass them in to be
  * initialized.
  * 
+ * resetParams: Does everything except (re)initialization of the parameters.
+ * Use when you want adjust a param and need to reset derived quantities, eg ssp.
+ * Should be called before finalize (or it will segfault on the unallocated parts).
+ * 
```

Comment:
Not sure I understand this comment--it seems like what you've implemented is:
- call setup(resetParams = true) to actually initialize
- manually change values
- call setup(resetParams = false)

Which is workable (but not ideal, see below comments), but the comment seems to say the opposite of this. Also what's this about calling setup before finalize?

Reply:
I'll improve the wording, and your interpretation was right.
Finalize deletes memory and you just can't call it if you want to modify parameters, i.e., with resetParams=false. Design issues aside, this comment needs clarification.